packages/design-picker/src/types.ts

```diff
@@ -93,6 +93,7 @@ export interface Design {
 	price?: string;
 	verticalizable?: boolean;
 	software_sets?: SoftwareSet[];
```

Comment:
I think we can now remove `software_sets` (at least for now) as it is not used anywhere :)